### PR TITLE
ISSUE-500: Changes related to enabling JWT based SSO along with SPNEG…

### PR DESCRIPTION
**Pull request by priyank5485**
…O/Kerberos

### CODE REVIEW

**Comment by priyank5485** [comment]
@omkreddy I have tested this on a kerberized and sso enabled cluster and it works. Can you please review it?

File: common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java [hunk 1]
```diff
@@ -368,4 +387,33 @@ protected boolean validateExpiration(SignedJWT jwtToken) {
...
+     * A User-Agent String is considered to be a browser if it does not contain
+     * any of the values from non-browser.user-agents; the default
+     * behavior is to consider everything a browser unless it contains one of:
+     * "java", "curl", "wget", "perl" or "Jersey".  Subclasses can optionally override
+     * this method to use different behavior.
+     *
+     * @param userAgent The User-Agent String, or null if there isn't one
+     * @return true if the User-Agent String refers to a browser, false if not
+     */
+    private boolean isBrowser(String userAgent) {
```
Comment (omkreddy) [inline common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java]: Can we avoid duplicating this method between handler classes? maybe we can move to util class?

File: common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java [hunk 2]
```diff
@@ -147,6 +149,14 @@ public void init(Properties config) throws ServletException {
         if (customCookieName != null) {
             cookieName = customCookieName;
         }
+
+        nonBrowserUserAgents = config.getProperty(
```
Comment (omkreddy) [inline common-auth/src/main/java/com/hortonworks/registries/auth/server/JWTAuthenticationHandler.java]: Can we avoid duplicating code between handler classes? maybe we can create  getNonBrowserUserAgents() util method?

**Comment by priyank5485** [comment]
@omkreddy Thanks for the review. I have addressed your review comments.

**Pull request merged**